Misc/NEWS.d/next/Library/2025-04-12-19-42-51.gh-issue-131913.twOx7K.rst

```diff
@@ -0,0 +1 @@
+add :func:`multiprocessing.Process.interrupt`
```

Comment:
This needs a better wording. In addition, a whatsnew/3.14.rst entry must be created as well.